public/sitemap-0.xml

```diff
@@ -1,24 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
-<url><loc>https://grabtern.com</loc><lastmod>2023-08-08T16:06:19.390Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/auth/forgotpass</loc><lastmod>2023-08-08T16:06:19.390Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
-<url><loc>https://grabtern.com/auth/login</loc><lastmod>2023-08-08T16:06:19.390Z</lastmod><changefreq>daily</changefreq><priority>0.7</priority></url>
```

Comment:
this file needs not to be changed...could you rever this and put this into gitignore!

Reply:
ok